hooks/usePositionHandler.tsx

```diff
@@ -0,0 +1,33 @@
+import {useState, useEffect} from 'react'
+
+export const usePositionHandler = () => {
+  const [positions, setPositions] = useState<{asset: string; amountOfLiquidity: string; amountOfCollateral: string; profit: string; loss: string}[]>()
+
+  useEffect(() => {
+    setPositions([
+      {
+        asset: 'ETH - BN',
+        amountOfLiquidity: '100',
+        amountOfCollateral: '200',
+        profit: '20',
+        loss: '80',
+      },
+      {
+        asset: 'ZIL - ETH',
+        amountOfLiquidity: '154',
+        amountOfCollateral: '375',
+        profit: '12',
+        loss: '88',
+      },
+      {
+        asset: 'DAI - BTC',
+        amountOfLiquidity: '45687',
+        amountOfCollateral: '1234',
+        profit: '100',
+        loss: '00',
+      },
+    ])
+  }, [])
+
```

Comment:
this might need to be changed later, maybe checking whether user is logged in to their metamask

Reply:
Yes for now this pr is for the design of position table. Once the contracts/factory is ready we will get the data from there and populate the fields based on the returned data